src/gromacs/nblib/gmxsetup.cpp

```diff
@@ -77,7 +78,7 @@ static Nbnxm::KernelType translateBenchmarkEnum(const BenchMarkKernels& kernel)
  *
  * Returns an error string when the kernel is not available.
  */
-static gmx::compat::optional<std::string> checkKernelSetup(const NBKernelOptions& options)
+static nonstd::optional<std::string> checkKernelSetup(const NBKernelOptions& options)
```

Comment:
Hopefully, this will change in the next few days: https://gitlab.com/gromacs/gromacs/-/merge_requests/98